Ad alerts and dashboard page to new quickstart guide (#13522)
getsentry/sentry-docs: docs/product/onboarding/alerts-dashboards.mdx

---
title: Sentry Alerts and Dashboards
sidebar_title: Alerts and Dashboards
description: "Get started with Sentry alerts and dashboards to monitor key metrics, track issue trends, and gain real-time insights into your application performance."
sidebar_order: 6000
---

By the end of this guide you'll be able to create and customize Sentry alerts and dashboards to monitor key metrics, track issue trends, and gain real-time insights in to your application performance. If you get stuck anywhere, try [asking Sentry AI](/?askAI=true) or join our [Discord](https://discord.gg/sentry) community.

## 1. Alerting Strategy

> Now that you've integrated with Slack and have added your members and teams, it's time to create [Sentry alerts](/product/alerts/best-practices/) to ensure real-time issue detection, enabling faster debugging and reducing downtime. Configuring alerts using specific thresholds or conditions helps prioritize problems, improving system reliability and incident response time.

Reviewer: I feel like I missed something here. I know we had a short spiel in the root doc about integrating with Slack, but we're not being very directive in that doc about integrating with any of those suggestions. It might be fine because we're expecting people to start with the quick guide then move on to here pretty close in time, but reading it like this, I feel like I missed a step. 

```suggestion
After [integrating with Slack](https://sentry-docs-git-ob-alerts.sentry.dev/organization/integrations/notification-incidents/slack/) and adding your members and teams, it's time to create [Sentry alerts](/product/alerts/best-practices/) to ensure real-time issue detection, enabling faster debugging and reducing downtime. Configuring alerts using specific thresholds or conditions helps prioritize problems, improving system reliability and incident response time.
```

### Alert Recommendations

- **Error Alert (High-Frequency)** - Triggers when a new or existing production issue suddenly spikes in frequency.

> <Arcade src="https://demo.arcade.software/YaLyaOnchFhnNgnnegdx?embed" />

Reviewer: This went just a little bit fast for me. I was able to follow along, but I couldn't easily paus any of it due to each step's short length.

Reviewer: Maybe it shouldn't auto-play? I think that was the problem, compared the the very next one.

### Recommended Dashboards

1. **Error Overview Dashboard** - Provides a high-level view of application health, showing you error trends, most frequent issues, and spikes in error occurrences.

2. **Performance Monitoring Dashboard** - Tracks transaction durations, slow API calls, and database query performance to identify bottlenecks.

3. **Mobile Crash Rate and Release Health Dashboard** - Monitors application stability and release impact by tracking crash rates across different versions.

4. **Critical Experience Dashboard** - Tracks errors or performance bottlenecks in your application's critical experiences. [See an example](https://sandbox.sentry.io/dashboard/126679/).

Congratulations on completing the Sentry onboarding guide! You now have the foundation to make the most out of Sentry. Here are some highlights on what you've implemented:

- **Projects and Integrations**: Connect your projects and integrate with your tools to track errors and performance.

- **Team Members**: Invite teammates and manage permissions for better collaboration.

- Adding Custom Context and performance instrumentation.

> - **Alerts**: Set up custom alerts to say on top of critical issues.

Reviewer: ```suggestion
- **Alerts**: Set up custom alerts to stay on top of critical issues.
```
